pipelines/botbuilder-python-functional-test-linux.yml

```diff
@@ -54,8 +52,6 @@ jobs:
       DockerNamespace: $(containerRegistry)
       DockerRepository: $(imageRepository)
       DockerImageTag: $(buildIdTag)
-      AppSettings: '-MicrosoftAppId $(botAppId) -MicrosoftAppPassword $(botAppPassword) -FLASK_APP /functionaltestbot/app.py -FLASK_DEBUG 1'
-
-      #StartupCommand: 'flask run --host=0.0.0.0 --port=3978'
-
+      AppSettings: '-MicrosoftAppId $(LinuxTestBotAppId) -MicrosoftAppPassword $(LinuxTestBotAppSecret) -FLASK_APP /functionaltestbot/app.py -FLASK_DEBUG 1'
 
+      #StartupCommand: 'flask run --host=0.0.0.0 --port=3978'
```

Comment:
Really a minor thing (you can ignore it if you want) but would you consider adding a break line here?

Reply:
Sure. New line added at EOF. Thanks.